processor/src/main/resources/org/mapstruct/ap/internal/model/assignment/SetterWrapperForCollectionsAndMaps.ftl

```diff
@@ -43,20 +43,16 @@
 -->
 <#macro callTargetWriteAccessor>
     <@lib.handleNullCheck>
-      <#if directAssignment>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( <@wrapLocalVarInCollectionInitializer/> );
-    <#else>
-        ${ext.targetBeanName}.${ext.targetWriteAccessorName}( ${localVarName} );
-    </#if>
```

Comment:
probably it does.. only In this case the whole thing fits on one line.. After all its the same call, only the arg differs.